include/swift/Basic/Malloc.h

```diff
@@ -44,4 +44,4 @@ inline void AlignedFree(void *p) {
   
 }
 
-#endif
+#endif /* __SWIFT_MALLOC_H__ */
```

Comment:
No worries. This is a learning process. How would you know otherwise if you are starting to contribute?